build_defs/internal_do_not_use/j2cl_application.bzl

```diff
@@ -61,6 +62,7 @@ def j2cl_application(
             "--rewrite_polyfills=%s" % rewrite_polyfills,
         ],
         deps = [":%s_config" % name] + deps,
+        **kwargs
```

Comment:
this is the full set of changes to `j2cl_application` (accepting `kwargs`, then passing them in to `js_binary`)